.nvmrc

```diff
@@ -1 +1 @@
-v16.15.0
\ No newline at end of file
+v18.14.2
```

Comment:
I think this file can be deleted since we have `.node-version`? @isBatak

Reply:
Yes